fighters/robot/src/acmd/specials.rs

```diff
@@ -338,9 +364,17 @@ unsafe extern "C" fn game_specialhirise(agent: &mut L2CAgentBase) {
 
     frame(lua_state, 1.0);
     if is_excute(agent) {
-        let mut damage = charge_frame / 3.0;
+        
+        if boma.is_prev_situation(*SITUATION_KIND_GROUND) { 
+            let mut damage = charge_frame / 1.5;
+        } else {
+            let mut damage = charge_frame / 3.0;
+        }
+
```

Comment:
builds fail because `damage` is no longer in-scope of the attack functions below

```rs
let damage = if boma.is_prev_situation(*SITUATION_KIND_GROUND) { 
    charge_frame / 1.5
} else {
    charge_frame / 3.0
};
```